5.cpp

```diff
@@ -1,6 +1,6 @@
 #include <iostream>
 #include <vector>
-#include "class5.h"
+#include "header_for_5_lab.h"
```

Comment:
reading name of heading, I have to imagine what's inside. header_for_5_lab say to me nothing